inbm/dispatcher-agent/dispatcher/dispatcher_broker.py

```diff
@@ -108,19 +103,22 @@ def send_result(self, message: str, request_id: str = "", job_id: str = "") -> N
                 message_dict = json.loads(message)
             except json.JSONDecodeError as e:
                 logger.error(f"Cannot convert formatted message to dict: {message}. Error: {e}")
```

Comment:
for a user-visible error message, should explain more about context of the error